stdlib/datetime.pyi

```diff
@@ -223,10 +223,13 @@ class datetime(date):
     def tzinfo(self) -> _tzinfo | None: ...
     @property
     def fold(self) -> int: ...
+    # The parameters in `fromtimestamp` are actually both positional-or-keyword parameters,
+    # but the first parameter is named "timestamp" in the C implementation and "t" in the Python implementation,
+    # so it is only truly *safe* to pass it as a positional argument.
     @classmethod
-    def fromtimestamp(cls: Type[_S], t: float, tz: _tzinfo | None = ...) -> _S: ...
+    def fromtimestamp(cls: Type[_S], __timestamp: float, __tz: _tzinfo | None = ...) -> _S: ...
```

Comment:
Why is `tz` positional-only? It seems to me that the Python implementation and the C implementation both accept `tz=blah`.

Reply:
Yes, sorry, I made a mistake there. Already fixed.